addon/components/polaris-choice-list.js

```diff
@@ -104,6 +121,22 @@ export default Component.extend({
    */
   onChange() {},
 
+  /**
+   * @private
+   */
+  hasError: bool('error'),
+
+  /**
+   * @private
+   */
+  finalNameError: computed('finalName', 'error', function() {
```

Comment:
Wondering if this prop name is a bit unclear - what do you think about using something like `ariaDescribedBy`?

Reply:
Good call, I'm going to also update `hasError` to `ariaInvalid` since it's only used for the `aria-invalid` attribute binding 👍